steps/mobitru-upload/1.0.0/step.yml

```diff
@@ -0,0 +1,104 @@
+title: Mobitru Upload
+summary: |
+  Upload an APK or IPA file to Mobitru platform
+description: |
+  Upload your application to [Mobitru](https://mobitru.com) and get back the app's artifact ID.
+  It can be used to install the app on Mobitru remote device for manual or automated testing.
+website: https://github.com/Mobitru/bitrise-step-mobitru-upload
+source_code_url: https://github.com/Mobitru/bitrise-step-mobitru-upload
+support_url: https://github.com/Mobitru/bitrise-step-mobitru-upload/issues
+published_at: 2024-05-27T11:42:06.933734+03:00
+source:
+  git: https://github.com/Mobitru/bitrise-step-mobitru-upload.git
+  commit: 78cd8386c8f729185bdb97359b9c0df377f14518
+project_type_tags:
+- ios
+- android
+- xamarin
```

Comment:
Please remove `xamarin` from the `project_type_tags` in the next Step versions, as [the official Xamarin support ended in 2022 at Bitrise](https://bitrise.io/blog/post/xamarin-support-ends-in-2022-on-bitrise).